### PR TITLE
App crash when open favorite album

### PR DESCRIPTION
**Pull request by HaiDang666**
Album`type` was set as 'unknown' before save to config. I also handle `error` state in AlbumView, it helps the case disconnect internet when requesting album info

### CODE REVIEW

File: packages/app/app/components/AlbumView/index.js [hunk 1]
```diff
@@ -175,18 +175,24 @@ class AlbumView extends React.Component {
...
+    const { t } = this.props;
     return (
       <div className={styles.album_view_container}>
         <Dimmer.Dimmable>
           <Dimmer active={album?.loading}>
             <Loader />
           </Dimmer>
           {
-            album?.loading !== true && (
+            album?.loading !== true && album?.title ? (
```
Comment (nukeop) [inline packages/app/app/components/AlbumView/index.js:186]: Can you change it to `!album?.loading && album?.title`?

File: packages/app/app/components/AlbumView/index.js [hunk 2]
```diff
@@ -175,18 +175,24 @@ class AlbumView extends React.Component {
...
               <div className={styles.album}>
                 {this.renderAlbumInfoBox(album, albumImage)}
                 {this.renderAlbumTracksList(album)}
               </div>
+            ) : album?.error === true && (
```
Comment (nukeop) [inline packages/app/app/components/AlbumView/index.js:191]: Please change to `            ) : !album?.loading && album?.error && (`

File: packages/i18n/src/locales/en.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
     "download": "Download",
     "download-toast-content": "{{artist}} - {{title}} has been added to downloads.",
     "download-toast-title": "Album added to downloads",
+    "error-load": "Ops, could not load this album. Try later :(",
```
Comment (nukeop) [inline packages/i18n/src/locales/en.json:6]: Please change the message to "Sorry, this album could not be loaded. Try again later or open a new issue on Github"

**Comment by HaiDang666** [comment]
resolved your suggestion

**Pull request merged**